src/pages/blog/starlingx-ptp-redundant-ptp-timing-clock-sources.md

```diff
@@ -0,0 +1,92 @@
+---
+templateKey: blog-post
+title: Redundant PTP timing clock sources
+author: Andre Mauricio Zelak
+date: 2024-03-25
+category:
+  - label: Features & Updates
+    id: category-A7fnZYrE1
+---
+In the rapidly evolving telecommunications landscape, 5G stands as a beacon of innovation, promising unparalleled connectivity and transformative capabilities. With its potential to revolutionize industries ranging from healthcare to manufacturing, the rollout of 5G networks has accumulated immense attention and anticipation. Beneath the surface of this technology lies an indispensable element, the Precision Time Protocol (PTP).
+
+And as part of the constant effort to meet the evolving needs of telecommunications industry and to provide a more robust Precision Time Protocol (PTP) deployment, the StarlingX 9.0 introduces the High Availability feature, providing a way to sync multiple Physical Hardware Clock (PHC) in a N+1 redundancy configuration.
+
+The feature allows, in a t-GM application, multiple GPS clock sources connected, and in a t-BC application, multiple PTP sources connected to redundant NICs, provinding protection against failtures.
```

Comment:
```suggestion
The feature allows, in a t-GM application, multiple GPS clock sources connected, and in a t-BC application, multiple PTP sources connected to redundant NICs, providing protection against failures.
```

Reply:
Can you please also briefly describe t-GM and t-BC applications? Or add a pointer to a good resource that does? That will make the blog post more accessible to a wider range of audience.

Reply:
Accepted the suggested changes and added a link to a PTP acronyms glossary.

Reply:
Done!